x/fungible/keeper/msg_server_deploy_fungible_coin_zrc20.go

```diff
@@ -37,7 +37,7 @@ func (k msgServer) DeployFungibleCoinZRC20(goCtx context.Context, msg *types.Msg
 	var address common.Address
 	var err error
 
-	if msg.Creator != k.observerKeeper.GetParams(ctx).GetAdminPolicyAccount(zetaObserverTypes.Policy_Type_deploy_fungible_coin) {
+	if msg.Creator != k.observerKeeper.GetParams(ctx).GetAdminPolicyAccount(zetaObserverTypes.Policy_Type_group2) {
```

Comment:
I saw in other two places the `Policy_Type_deploy_fungible_coin` were replaced by `Policy_Type_group1`. Was this a typo?

Reply:
There are some were it was replaced with 1 and other 2.
Deploy and remove can potentially be financially exploited, therefore those are group 2